app/models/tasks/copy_media_task.rb

```diff
@@ -112,7 +112,12 @@ def porter_slice_task
       Destination: {
         Mode: "AWS/S3",
         BucketName: ENV["FEEDER_STORAGE_BUCKET"],
-        ObjectKey: porter_escape(media_resource.path)
+        ObjectKey: porter_escape(media_resource.path),
+        ContentType: "REPLACE",
```

Comment:
Since ffmpeg is just slicing the file, not changing the format, the content type of the sliced file should match the input (uncut) file.